src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.2.1'
+ModuleVersion = '1.3.1'
```

Comment:
If you are updating this, you should also update AssemblyInfo.cs

Reply:
fixed， thanks!